src/clients/mdw-client.model.ts

```diff
@@ -37,3 +38,19 @@ export type BalancesV1 = {
   contract_id: string;
   height: string;
 };
+
+// TODO remove unused fields?
```

Comment:
Disagree, we should represent the data here as is to avoid giving the suggestion that the response is smaller.